### DISCUSSION TITLE
Mnemonic Phrase Creation and BIP-39 Standards

### DISCUSSION THREAD

**Discussion by aswanthabam** [xclud/web3dart#134] [General]
I'm trying to figure out how to create mnemonic phrases using your library.

Also, does it follow the BIP-39 standards?

**Comment by xclud** [comment]
Hi, my other package does:

https://pub.dev/packages/wallet

**Reply by aswanthabam** [reply]
Thanks: } Can you explain why the feature can't be added to this package?
